### PR TITLE
Increase the Arkansas capital gains loss cap parameter value and edit the AR income tree to account for separate income calculations

### LINKED ISSUE #4443: Fix Arkansas capital gain loss cap parameter values
The 2021 AR1000D form says this in RED:

Screenshot 2024-04-23 at 3 45 42 PM

.
.

But the following `loss_cap.yaml` file has different values:

https://github.com/PolicyEngine/policyengine-us/blob/c64a7a20ac62a8544df3c45697d9842ca4cdefb1/policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml#L1-L29

.
.
As a result, Policyengine-US 0.749.1 fails the following simple unit test:

```
- name: AR taxable capital gains unit test 2
  absolute_error_margin: 0.01
  period: 2021
  input:
    filing_status: SINGLE
    long_term_capital_gains: -4_000
    state_fips: 5  # AR
  output:  # expected results from filing out 2021 AR1000D form
    ar_taxable_capital_gains: -3_000
```

### PR DESCRIPTION
**Pull request by PavelMakarchuk**
Fixes #4443

### CODE REVIEW

**Comment by PavelMakarchuk** [comment]
While working on this we will probably need to make the following adjustments:

Split ar_gross_income into indiv and joint as well as ar_agi as there is a different cap for married filers filing separately on the same return

File: policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml [hunk 1]
```diff
@@ -17,13 +17,12 @@ metadata:
       href: https://www.taxformfinder.org/forms/2023/2023-arkansas-form-ar1000d.pdf#page=1
```
Comment (MaxGhenis) [inline policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml:17]: Leave a comment mapping the numeric filing statuses

**Review by MaxGhenis — APPROVED** [review]
good to merge after this

File: policyengine_us/parameters/gov/states/ar/tax/income/gross_income/sources/individual.yaml [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  2021-01-01:
+    - irs_employment_income # Line 8
+    - self_employment_income # Line 8
+    - military_service_income # Line 9
+    - interest_income # Line 10
+    - dividend_income # Line 11
+    - alimony_income # Line 12
+    - ar_taxable_capital_gains_indiv # Line 13
+    - taxable_ira_distributions # Line 16
+    # - ar_military_retirement_income_person # Line 17
```
[the remark this answers is not in the document]
Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/gross_income/sources/individual.yaml]: Added the relevant income instead - same for joint

File: policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml (hunk 1 above)
Comment (MaxGhenis) [reply policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml:17]: what's 5?

Comment (PavelMakarchuk) [reply policyengine_us/parameters/gov/states/ar/tax/income/gross_income/capital_gains/loss_cap.yaml:17]: Married filing separately on separate returns.

**Pull request merged**